### PR TITLE
[zh-CN] sync Notification.body with en-US

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

sync Notification.body with en-US

### Motivation

sync Notification.body with en-US

### Additional details

https://developer.mozilla.org/en-US/docs/Web/API/Notification/body
https://developer.mozilla.org/zh-CN/docs/Web/API/Notification/body

### CODE REVIEW

File: files/zh-cn/web/api/notification/body/index.md [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
-{{domxref("Notification")}} 的只读属性 body 返回了构造函数{{domxref("Notification")}}实例化时，options 所使用的 body 值。
-
-{{AvailableInWorkers}}
-
-## 语法
-
-```plain
-var body = Notification.body;
-```
+{{domxref("Notification")}} 接口的只读属性 **`body`** 返回了构造函数 {{domxref("Notification.Notification","Notification()")}} 实例化时参数所使用的 body 值。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/notification/body/index.md:8]: 与原文不太一致
https://github.com/mdn/content/blob/4ffa87827a723576cd3bc75ad2b84bd7e3e544d9/files/en-us/web/api/notification/body/index.md?plain=1#LL11-L14

**Review by jasonren0403 — DISMISSED** [review]
Just one nit

File: files/zh-cn/web/api/notification/body/index.md [hunk 2]
```diff
@@ -1,35 +1,27 @@
...
-title: Notification.body
+title: Notification：body 属性
 slug: Web/API/Notification/body
 ---
 
-{{APIRef("Web Notifications")}}
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
 
-{{domxref("Notification")}} 的只读属性 body 返回了构造函数{{domxref("Notification")}}实例化时，options 所使用的 body 值。
+{{domxref("Notification")}} 接口的 **`body`** 只读属性指示通知的正文字符串，在 {{domxref("Notification.Notification","Notification()")}} 构造函数的 `body` 选项中指定。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/body/index.md:8]: Let's make it more specified ;)

```suggestion
{{domxref("Notification")}} 接口的 **`body`** 只读属性指示通知的正文字符串，该字符串在 {{domxref("Notification.Notification","Notification()")}} 构造函数的 `body` 选项中指定。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**